new module

```diff
@@ -0,0 +1 @@
+developed funtionalty for redbus module one
```

Comment:
please correct the functionality spelling.

Reply:
corrected.